src/utils/get_searched_result/get_searched_result.js

```diff
@@ -0,0 +1,17 @@
+export const get_searched_result = (searchValue, product_list) => {
+  searchValue = searchValue.trim();
+  const searched_result = [];
+  if (searchValue.length === 0) {
+    return searched_result;
+  } else {
+    product_list.forEach((element) => {
```

Comment:
You can use `.filter` rather than `forEach` it will improve your array immutability .

Reply:
Actually i am generating every time a new array when this function is called..so i preferred this..